elastic/elastic-agent
# Remove PGP signature verification skip for DEV builds

## What does this PR do?

Remove PGP signature verification skip for DEV builds

## Why is it important?

It's not necessary anymore and prevents the PGP signature verification being skipped on a production build if it is ever, by mistake, produced with `DEV=true`.

## Checklist




## How to test this PR locally

Produce a build with `DEV=true`, upgrade (actually downgrade it) it, check there is a log like `Verification with PGP[%d] successful|faied`

## Related issues

- Closes #3589 


## Logs
TODO

## Questions to ask yourself

- How are we going to support this in production? 
- How are we going to measure its adoption? 
- How are we going to debug this?
- What are the metrics I should take care of?
- ...
